documentation/docs/installation.md

```diff
@@ -15,7 +15,7 @@ See the steps below to get started and install the Serverless Nextjs component i
 # serverless.yml
 
 myNextApplication:
-  component: "@sls-next/serverless-component@1.15.2-alpha.1" #specify latest stable version
+  component: "@sls-next/serverless-component@1.18" #specify latest stable version
```

Comment:
1.19.0 is the latest, also maybe just add a comment to refer to https://www.npmjs.com/package/@sls-next/serverless-component for the latest stable version?